Codeinwp/neve
# Style issues on WooCommerce Cart and Checkout pages

### Summary


### Will affect visual aspect of the product

YES

### Test instructions


- Should fix the visual issues mentioned in #627 
- You can simply copy-paste the referenced site HTML markup inside your local WP instance and the issues should be fixed.


Closes #627.
